test/unit/client/test_client.py

```diff
@@ -387,6 +424,31 @@ def test_create_enterprise_user_returns_the_correct_user_object(mock_client, moc
     assert new_user.name == test_user_name
 
 
+def test_terms_of_service_returns_the_correct_tos_object(mock_client, mock_box_session, terms_of_service_response):
```

Comment:
The test name should match up with the method under test (`create_terms_of_service` here)